presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -125,6 +126,15 @@ protected void cleanUpSchemas(List<String> schemaNames)
         }
     }
 
+    @Override
+    protected TestTable createTableWithDefaultColumns()
+    {
+        return new TestTable(
+                sqlServer::execute,
+                "table",
+                "(a bigint, b bigint default 40)");
```

Comment:
Add two more columns: 
```suggestion
                "(a bigint, b bigint default 40, c bigint, d bigint)");
```

goal:

1. treat `c` as a default column (with _implicit_ `NULL` default)
2. in test, treat `d` as a obligatory column that you insert into (along inserting to `a`).
   This is so that you verify that inserting to 1st and 4th column doesn't get shifted (eg data could land in 1st and 2nd column instead)

Reply:
same for all other test classes

Reply:
Actually his should have these columns:

* `col_required NOT NULL` -- required
* `col_nullable` -- not required
* `col_default DEFAULT  43` -- not required
* `col_nonnull_default NOT NULL DEFAULT 42` -- not null, but has default, not required
* `col_required2 NOT NULL` -- required; additional required column to ensure data isn't shifted on INSERT.

Now, if you pick up a type that is supported by any reasonable database (eg `varchar(5)`), you can have same schema in every subclass.
Even if you do not define the schema once, your test can verify the schema is as expected (verify that all subclasses are updated the same way).